docs/version0.9.md

```diff
@@ -135,6 +137,93 @@ For performance and compatibility with the new String object layout, the OpenJ9
+- [JEP 254](https://openjdk.org/jeps/254): Compact Strings (not enabled by default)
+- [JEP 255](https://openjdk.org/jeps/255): Merge Selected Xerces 2.11.0 Updates into JAXP
+- [JEP 256](https://openjdk.org/jeps/256): BeanInfo Annotations
+- [JEP 257](https://openjdk.org/jeps/257): Update JavaFX/Media to Newer Version of GStreamer
+- [JEP 258](https://openjdk.org/jeps/258): HarfBuzz Font-Layout Engine
+- [JEP 259](https://openjdk.org/jeps/259): Stack-Walking API
+- [JEP 260](https://openjdk.org/jeps/260): Encapsulate Most Internal APIs
+- [JEP 261](https://openjdk.org/jeps/261): Module System
+- [JEP 262](https://openjdk.org/jeps/262): TIFF Image I/O
+- [JEP 263](https://openjdk.org/jeps/263): HiDPI Graphics on Windows and Linux®
+- [JEP 264](https://openjdk.org/jeps/264): Platform Logging API and Service
+- [JEP 265](https://openjdk.org/jeps/265): Marlin Graphics Renderer
+- [JEP 266](https://openjdk.org/jeps/266): More Concurrency Updates
+- [JEP 267](https://openjdk.org/jeps/267): Unicode 8.0
+- [JEP 268](https://openjdk.org/jeps/268): XML Catalogs
+- [JEP 269](https://openjdk.org/jeps/269): Convenience Factory Methods for Collections
+- [JEP 272](https://openjdk.org/jeps/272): Platform-Specific Desktop Features
+- [JEP 273](https://openjdk.org/jeps/273): DRBG-Based SecureRandom Implementations
+- [JEP 274](https://openjdk.org/jeps/227): Enhanced Method Handles
+- [JEP 275](https://openjdk.org/jeps/275): Modular Java Application Packaging
+- [JEP 276](https://openjdk.org/jeps/276): Dynamic Linking of Language-Defined Object Models
+- [JEP 277](https://openjdk.org/jeps/277): Enhanced Deprecation
+- [JEP 280](https://openjdk.org/jeps/280): Indify String Concatenation
+- [JEP 282](https://openjdk.org/jeps/282): jlink: The Java Linker
+- [JEP 283](https://openjdk.org/jeps/283): Enable GTK 3 on Linux
+- [JEP 285](https://openjdk.org/jeps/285): Spin-Wait Hints
+- [JEP 287](https://openjdk.org/jeps/287): SHA-3 Hash Algorithms
+- [JEP 288](https://openjdk.org/jeps/288): Disable SHA-1 Certificates
+- [JEP 289](https://openjdk.org/jeps/289): Deprecate the Applet API
+- [JEP 290](https://openjdk.org/jeps/290): Filter Incoming Serialization Data
+- [JEP 292](https://openjdk.org/jeps/292): Implement Selected ECMAScript 6 Features in Nashorn
+- [JEP 298](https://openjdk.org/jeps/298): Remove Demos and Samples
+- [JEP 299](https://openjdk.org/jeps/299): Reorganize Documentation
+
+You can find the full list of features for JDK 9 at the [OpenJDK project](http://openjdk.java.net/projects/jdk9/). Any remaining features that are listed do not apply to OpenJ9.
+
+### ![Start of content that applies to Java 10 plus](cr/java10plus.png) New JDK 10 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 286](https://openjdk.org/jeps/286): Local-Variable Type Inference
+- [JEP 296](https://openjdk.org/jeps/296): Consolidate the JDK Forest into a Single Repository
```

Comment:
This one shouldn't be listed, it has nothing to do with the runtime, but how the OpenJDK source code is organized.